src/main/main.cpp

```diff
@@ -48,8 +54,13 @@ int main(int argc, char *argv[])
 
     QGuiApplication app(argc, argv);
 
+    #ifdef Q_OS_MACOS
+    MacOsEventListener evListener;
+    initMacOsEventListener(&evListener);
```

Comment:
@JohnPoison then we need to unindent the other ones.
I also indented all `switch` statements even though QtCreator doesn't because I think it both cases it's easier to read. But it's more important that we're consistent, so I have no problem to change it in both cases.